### DISCUSSION TITLE
Project 4: 2D visualization for small molecules

### DISCUSSION THREAD

**Discussion by Pavan-Satyam** [MDAnalysis/mdanalysis#4496] [GSoC Discussions]
@cbouy 
@hmacdope 
@xhgchen 

Hi Team,
I want to work on this project.Could you guide what are the next steps to get view on the code?

**Comment by hmacdope** [comment]
Check out the [UserGuide](https://userguide.mdanalysis.org/stable/index.html):  and the GSOC2024 projects [page](https://github.com/MDAnalysis/mdanalysis/wiki/GSoC-2024-Project-Ideas)

**Reply by abhishekd358** [reply]
hello ,My name is Abhishek , final year student and i have playing with python past 2 years. As mentioned above I gone through MD analysis Documentations, also learn MD analysis. May I know what should I do to get involve with the project

@cbouy
@hmacdope
@xhgchen

**Reply by hmacdope** [reply]
The information is literally in the comment above yours.

**Comment by Pavan-Satyam** [comment]
Hi, @hmacdope I went through the code ,I have make some understanding of md analysis and rdkit.I am struck where and how to give preproposal and What you are expecting as endobjective from project 4.

**Comment by Pratibhabhalerao080** [comment]
hello, My name is Pratibha, I am a student doing my master's in Data Science. I would like to know how can I participate in this project.

**Reply by hmacdope** [reply]
Read up about 3 lines.
